To: vim_dev@googlegroups.com
Subject: Patch 7.3.132
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.132
Problem:    C++ style comments.
Solution:   Change to C comments.
Files:	    src/if_python3.c


*** ../vim-7.3.131/src/if_python3.c	2010-11-16 19:25:56.000000000 +0100
--- src/if_python3.c	2011-01-16 01:28:35.000000000 +0100
***************
*** 22,29 ****
   * Adaptations to support both python3.x and python2.x
   */
  
! // uncomment this if used with the debug version of python
! // #define Py_DEBUG
  
  #include "vim.h"
  
--- 22,29 ----
   * Adaptations to support both python3.x and python2.x
   */
  
! /* uncomment this if used with the debug version of python */
! /* #define Py_DEBUG */
  
  #include "vim.h"
  
***************
*** 74,80 ****
  #define PyString_Size(obj) PyUnicode_GET_SIZE(obj)
  #define PyString_FromString(repr) PyUnicode_FromString(repr)
  
! #if defined(DYNAMIC_PYTHON3)
  
  # ifndef WIN3264
  #  include <dlfcn.h>
--- 74,80 ----
  #define PyString_Size(obj) PyUnicode_GET_SIZE(obj)
  #define PyString_FromString(repr) PyUnicode_FromString(repr)
  
! #if defined(DYNAMIC_PYTHON3) || defined(PROTO)
  
  # ifndef WIN3264
  #  include <dlfcn.h>
*** ../vim-7.3.131/src/version.c	2011-02-25 15:17:14.000000000 +0100
--- src/version.c	2011-02-25 15:18:18.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     132,
  /**/

-- 
        **  Hello and Welcome to the Psychiatric Hotline **
If you are obsessive-compulsive, please press 1 repeatedly.
If you are co-dependent, please ask someone to press 2.
If you have multiple personalities, please press 3, 4, 5 and 6.
If you are paranoid-delusional, we know who you are and what you want
       - just stay on the line so we can trace the call.
If you are schizophrenic, listen carefully and a little voice will
       tell you which number to press next.
If you are manic-depressive, it doesn't matter which number you press
       - no one will answer.
If you suffer from panic attacks, push every button you can find.
If you are sane, please hold on - we have the rest of humanity on the
        other line and they desparately want to ask you a few questions.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
